package/yast2.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Thu Jun 17 06:15:02 UTC 2021 - Dirk Müller <dmueller@suse.com>
+
+- add riscv64 architecture helper
```

Comment:
in general change looks good. Just please add jira entry or bugzilla number here to clarify why change is needed and please also bump version, so it gets auto submitted.
Thanks

Reply:
done